api/service/types.ts

```diff
@@ -56,23 +56,23 @@ export interface DudewhereBucket {
 
 export interface StoreTable {
   exists: (space: DID, link: AnyLink) => Promise<boolean>
-  insert: (item: StoreItemInput) => Promise<StoreItemOutput>
+  insert: (item: StoreAddInput) => Promise<StoreAddOutput>
   remove: (space: DID, link: AnyLink) => Promise<void>
-  list: (space: DID, options?: ListOptions) => Promise<ListResponse<StoreListResult>>
+  list: (space: DID, options?: ListOptions) => Promise<ListResponse<StoreListItem>>
 }
 
 export interface UploadTable {
   exists: (space: DID, root: AnyLink) => Promise<boolean>
-  insert: (item: UploadItemInput) => Promise<UploadItemOutput[]>
+  insert: (item: UploadAddInput) => Promise<UploadAddResult>
```

Comment:
Should we follow store pattern and use Output for return value?

Reply:
we've not got a consistent pattern yet. The store table abstraction doesn't have enough info to create the `StoreAddResult` as the service includes other field like status and the signed url... so we go 

`StoreAddInput -> StoreAddOutput -> StoreAddResult`

whereas in for `upload/add` we already have all the info for the service level result. I'll split those types out to match if we need it.